requirements.txt

```diff
@@ -29,21 +28,21 @@ google-auth==2.23.4
 google-cloud-aiplatform[pipelines]==1.34.0
 google-cloud-firestore==2.13.1
 google-cloud-storage==2.13.0
-gunicorn==22.0.0
+gunicorn==23.0.0
 httplib2==0.22.0
-idna>=3.7 # not directly required, pinned by Snyk to avoid a vulnerability
+idna>=3.7               # not directly required, pinned by Snyk to avoid a vulnerability
 locust==2.32.3
 nameparser==1.1.3
 nltk>=3.9               # recommend engine - comment out if initial installation is failing
 numpy~=1.26.2           # recommend engine - comment out if initial installation is failing
 oauth2client==4.1.3
 oauthlib==3.2.2
-pandas~=2.1.3           # recommend engine - comment out if initial installation is failing
+pandas~=2.2.3
```

Comment:
All of the packages labeled "# recommend engine - comment out if initial installation is failing" were added for WeVoteServer/politician/controllers_recommendation.py a machine learning system which was created by a UC Berkeley Masters student. We have had some other engineers run into problems installing the packages required for that system to work, so we keep this label in place so we know what to comment out if a new engineer's installation is blocked.